esp/templates/email/default_email_html.txt

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
I think this is redundant -- we tell users to include their own `<html>` if they want HTML.